app.rb

```diff
@@ -0,0 +1,63 @@
+require_relative './teacher'
+require_relative './student'
+require_relative './rental'
+require_relative './book'
+require_relative './person'
+
+class App
+  def initialize
+    @books = []
+    @peoples = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: #{book.title} Author: #{book.author} \n" }
+  end
+
+  def list_peoples
+    @peoples.each do |people|
+      puts "[#{people.class.name}] Name: #{people.name} ID: #{people.id} Age: #{people.age} \n"
+    end
+  end
+
+  def create_teacher(age, specialization, parent_permission, name)
+    @peoples << Teacher.new(age, specialization, parent_permission: parent_permission, name: name)
+  end
+
+  def create_student(age, name, parent_permission)
+    @peoples << Student.new(age, name, parent_permission: parent_permission)
+  end
+
+  def create_book
+    puts 'Title:'
+    title = gets.chomp
+    puts 'Author:'
+    author = gets.chomp
+    @books << Book.new(title, author)
+  end
```

Comment:
- [ ] @MuhammadUsmanAslam Great work on your app. However am unable to add a new book, am getting the error below 
![image](https://github.com/MuhammadUsmanAslam/OOP-school-library/assets/61408860/68b2b0c3-d348-4f5b-a643-4d78e2358fa9)